.github/workflows/main.yml

```diff
@@ -21,3 +21,4 @@ jobs:
           mix deps.get
 
       - run: mix test
+      - run: EXQLITE_INTEGRATION=true mix test
```

Comment:
I was going to suggest fanning this one out to be more parallel, but I think this is a good first step.